www/js/metrics.js

```diff
@@ -443,6 +456,11 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
           var aggCount = results[2].aggregate_metrics;
```

Comment:
concretely, now that we check timestamps, if the user has no recent data, we will get an empty array anyway